src/app/providers/index.tsx

```diff
@@ -18,13 +20,17 @@ const queryClient = new QueryClient({
 
 export function Provider() {
   return (
-      <QueryClientProvider client={ queryClient }>
-        <BrowserRouter>
-          <Suspense fallback={ <FullPageWrapper><Spinner/></FullPageWrapper> }>
-            <Router/>
-          </Suspense>
-        </BrowserRouter>
-        <ReactQueryDevtools initialIsOpen={false} />
-      </QueryClientProvider>
+      <BrowserRouter>
+        <QueryClientProvider client={ queryClient }>
+          <StoreProvider>
+            <AuthProvider>
```

Comment:
you already use tanstack
are you sure that context providers still needed?
just example that I found https://dev.to/this-is-learning/react-query-authentication-flow-id2